data/netcdf_data.py

```diff
@@ -65,13 +65,18 @@ def __enter__(self):
                     # xarray won't open FVCOM files due to dimension/coordinate/variable label
                     # duplication issue, so fall back to using netCDF4.Dataset()
                     self.dataset = netCDF4.MFDataset(self._nc_files)
+
+            elif (self.url.endswith(".zarr") if not isinstance(self.url, list) else False):
+                ds_zarr = xarray.open_zarr(self.url, decode_times=decode_times)
+                self.dataset = ds_zarr
+
```

Comment:
I'm sorry to have to say this, @CubeCode93 , but the changes I made in PR #789 that got merged this week are in this area of the code, so you will have to merge with HEAD again in you branch. It might be a bit messy for you, I'm afraid. Sorry.

Reply:
Thanks for the review and comments @douglatornell 
That's totally fine and I have updated my branch, hope it's fine now!